Dockerfile.web

```diff
@@ -1,7 +1,7 @@
 ARG img_version
 FROM godot-fedora:${img_version}
 
-ENV EMSCRIPTEN_VERSION=3.1.53
+ENV EMSCRIPTEN_VERSION=3.1.61
```

Comment:
We'll want to bump to 3.1.62 as soon as it's released, and maybe aim to release 4.3 with that if tests are conclusive?

CC @adamscott @Faless

Reply:
Yeah, I think 62 will most likely be released before beta3, I don't think it's worth rolling it back (39 isn't perfect either).

Worst case scenario we can apply the needed patch after installing 61.

We could also try politely asking upstream for a new release next week :sweat_smile:

Reply:
> Worst case scenario we can apply the needed patch after installing 61.

I did that for now, I don't want to rush upstream needlessly. I complain enough when they make a release that breaks our build so if they need time to finalize 3.1.62, I'll give it to them :D 

https://github.com/godotengine/build-containers/pull/142